docs/docs/spark-procedures.md

```diff
@@ -936,3 +936,40 @@ as an `UPDATE_AFTER` image, resulting in the following pre/post update images:
 |-----|--------|--------------|
 | 3   | Robert | UPDATE_BEFORE|
 | 3   | Dan    | UPDATE_AFTER |
+
+## Table stats
```

Comment:
Sorry I just read it one more time, I think we should change this.  I generaly agree with @RussellSpitzer comment https://github.com/apache/iceberg/pull/11606#pullrequestreview-2449581358 to avoid abbreviation in the spec unless we are referring to the procedure name.